plugins/net_plugin/CMakeLists.txt

```diff
@@ -5,3 +5,7 @@ add_library( net_plugin
 
 target_link_libraries( net_plugin chain_plugin producer_plugin appbase fc )
 target_include_directories( net_plugin PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${CMAKE_CURRENT_SOURCE_DIR}/../chain_interface/include  "${CMAKE_CURRENT_SOURCE_DIR}/../../libraries/appbase/include")
+
+# once tests are added to the test subdirectory, the following line should be uncommented
+# add_subdirectory(test)
```

Comment:
The boost unit test environment requires that there be at least 1 unit test.  Otherwise, the unit test fails indicating no tests have been defined.  I debated adding a dummy init test so it would pass.  Once a unit test has been added, this comment will no longer be necessary and will be removed.  I did it this way so we could concentrate on the unit test framework and not on my unit test.  However, I am happy to add my unit test to this PR so we can review both the framework and my unit test at the same time.

Reply:
ok, just wanted to make sure